HTML/mockup-mogo/sass/base/_typography.scss

```diff
@@ -0,0 +1,170 @@
+.logo {
+    font-size: 3rem;
+    font-weight: bold;
+    text-align: left;
+    color: $color-main-light;
+
+    & a {
```

Comment:
hm if it should transpile to .logoa or a.logo than better without space, if it is just nesting and result .logo a you don't need `&` here and below and for other tags